.github/workflows/coverity.yml

```diff
@@ -0,0 +1,37 @@
+name: Coverity Scan
```

Comment:
This file should also be removed in our branch.

Reply:
Planning on removing the entire .github directory.  Also note that, per the new README, we'll never push the button here for merges, because github can't do it without removing the whole merge history tree (see TRITON-2354).